client/jetpack-cloud/components/layout/header.tsx

```diff
@@ -39,8 +39,36 @@ export default function LayoutHeader( { showStickyContent, children }: Props ) {
 
 	const outerDivProps = divRef ? { ref: divRef as React.RefObject< HTMLDivElement > } : {};
 
+	const [ minHeaderHeight, setMinHeaderHeight ] = useState( 0 );
+
+	// To avoid shifting the layout when displaying sticky content,  we will need to
+	// keep track of our Header height and set it as the minimum viewport height.
+	useEffect(
```

Comment:
I'm think if we can make use of `useLayoutEffect` here. WDYT?

Reply:
Yes, I agree. make sense 👍